tests/oss-fuzz/build.sh

```diff
@@ -42,15 +42,15 @@ export ORIG_CXXFLAGS="$CXXFLAGS"
 export CFLAGS=""
 export CXXFLAGS=""
 
-# fuzz flags are problematic with meson (hence no dav1d) and no point in fuzzing fuzztest.
-cd ext && bash dav1d.cmd && bash fuzztest.cmd && cd ..
+# Use normal flags for projects built outside of libavif (e.g. dav1d with meson is
+# problematic with sanitizer flags)
+cd ext && bash dav1d.cmd && bash libsharpyuv.cmd && bash libyuv.cmd && cd ..
 
 export CFLAGS=$ORIG_CFLAGS
 export CXXFLAGS=$ORIG_CXXFLAGS
 
 # build dependencies
-cd ext && bash aom.cmd && bash libjpeg.cmd && bash libsharpyuv.cmd && \
-      bash libyuv.cmd && bash zlibpng.cmd && cd ..
+cd ext && bash aom.cmd && bash fuzztest.cmd && bash libjpeg.cmd && bash zlibpng.cmd && cd ..
```

Comment:
Move the "build dependencies" comment before line 39 ("# Reset compile flags to build some deps without fuzzer flags."). Or update this comment to say that these dependencies are built as subprojects of libavif.

Reply:
I rephrased that and actually have dav1d be on its own because of meon.